src/ui/public/notify/notifier.js

```diff
@@ -97,10 +97,14 @@ function add(notif, cb) {
     });
   } else if (notif.customActions) {
     // wrap all of the custom functions in a close
-    notif.customActions = notif.customActions.map(action => {
+    notif.customActions = notif.customActions.map((action, idx) => {
```

Comment:
OH! _duh._ Can we rename it to `index`, then? I had no idea that `idx` was short hand... I thought it was some special kind of ID!